doc/source/theory/theory.rst

```diff
@@ -39,38 +27,16 @@ Multiphysics
 
     multiphysics/multiphysics
 
-Heat transfer
-----------------
-
-**Under construction**
-
-
-Advection-diffusion of a passive tracer
-----------------------------------------
-**Under construction**
-
-Discrete Element Method
+Multiphases
 =============================
-.. toctree::
-    :maxdepth: 3
-    :glob:
-    :titlesonly:
-
-    dem/dem
-
 
-Unresolved CFD-DEM
-=============================
 .. toctree::
     :maxdepth: 3
     :glob:
     :titlesonly:
 
-    unresolved_cfd-dem/unresolved_cfd-dem
+    multiphases/multiphases
```

Comment:
```suggestion
    multiphase/multiphase
```

There should be other modifications because of the change of the name of the section